### DISCUSSION TITLE
Some OpenCanary scripts to share

### DISCUSSION THREAD

**Discussion by SecuriLee** [thinkst/opencanary#326] [General]
I have some scripts running on my OC instances:
## process-monitor.sh
 - watches the opencanaryd/twistd process, it sometimes borks.  Sends webhook if that happens
## folder-watcher.sh
 - watches your open SMB share, copies files that are dropped there, submits to VirusTotal
## oc-monitor.sh
 - does a loopback to see if telnet is working (avoids logging) and will try to restart the process and finally reboot if it does not work

https://github.com/orgs/TheOpenCanaryExperience/repositories

**Comment by SecuriLee** [comment]
I've added an updated vnc.py which will avoid this: <Password was not in the common list>
It now contains around 1000 common passwords.

https://github.com/TheOpenCanaryExperience/Scripts-Tips-and-Tricks/blob/main/modules/vnc.py

(I hope this is not a performance killer, I now run with it on 3 OCs facing the Internet)

**Reply by SecuriLee** [reply]
Visualisation picking up in Splunk for VNC password guesses :)
![image](https://github.com/thinkst/opencanary/assets/117658191/d987ccf4-dfa5-4f2c-a896-1d5018e75a14)

**Comment by jayjb** [comment]
Hi @SecuriLee,

I completely missed this. It really looks very cool. Thanks for bringing it up. 1000 passwords sounds reasonable. 

Re: the script. They sound very useful! Let me see if we could include them as an operational addition of sorts.
